GoogleCombined/GoogleProject/LocationImageGenerator.swift

```diff
@@ -0,0 +1,99 @@
+/* Copyright (c) 2020 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import UIKit
+import GoogleMaps
+import GooglePlaces
+
+class LocationImageGenerator {
+    
+    /// This should be the dimension of the image generated, regardless of the iPhone model.
+    private let dim: Double = 110
+    
+    // MARK: Image lookup and placement methods
+    
+    /// Sets a marker's icon to a place's image, if it has one
+    ///
+    /// - Parameters:
+    ///   - placeId: The placeId of the location we wish to find an image of.
+    ///   - localMarker: The marker that we want to set the image on.
+    ///   - imageView: The image view that we want to set the image on.
+    ///   - select: Indicates if we want the image on the image view; if this is false, tapped should be true.
+    ///   - tapped: Indicates if we want the image on the GMSMarker; if this is false, select should be true.
+    ///   - width: The width of the image; it is set to default at 110.
```

Comment:
Can you provide some reason of setting 110 as default. Is it a random number, or with some consideration?

Reply:
Yep, I added a comment as to why I chose 110. It was found via trial and error among all different phone sizes.